### ISSUE TITLE
New user role selection not being saved on signup.

### ISSUE THREAD

**Issue by coldridj** [e-ucm/rage-analytics#26]
Clean analytics server installation. 
New user (developer) created. 
After logging in various API calls calls fail with HTTP error 403 in different sessions

/game/public

![image](https://cloud.githubusercontent.com/assets/18266637/19961517/ec6a79b4-a1aa-11e6-9b42-51e9c42c8156.png)

/api/roles

![image](https://cloud.githubusercontent.com/assets/18266637/19961521/f3ce719c-a1aa-11e6-9623-f9ccf4217a32.png)

**Comment by coldridj** [comment]
Upon further investigation, the developer role is not being assigned to the new user reliably. When added via the admin user management screen this issue is no longer present.

**Comment by coldridj** [comment]
This should not be closed, I can consistently repeat this error. User roles are not present on user after signup via /api/proxy/afront/signup - role selection can be seen in POST data.

POST: /api/signup {"username":"Test4","email":"jack+test4@playgen.com","password":"password","role":"developer","prefix":"gleaner"}

![image](https://cloud.githubusercontent.com/assets/18266637/20101986/3a348792-a5bc-11e6-949a-7f54a1ceaccd.png)

**Comment by gorco** [comment]
I forgot to comment when I closed the issue.

The user with developer role doesn't have permissions for /game/public (this is intentioned like https://github.com/e-ucm/rage-analytics/issues/34)
And the same with /api/roles route (only the admin have permissions to see the roles list)

Both are  design decisions.

So the problem is like https://github.com/e-ucm/rage-analytics/issues/34 (I closed it for that reason, the problem is the same) We'll remove the console log and the call for the developer in a future, but for now is usefull for us.
